bookkeeper-server/src/main/java/org/apache/bookkeeper/meta/MetadataBookieDriver.java

```diff
@@ -88,6 +89,13 @@ default CompletableFuture<Void> disableHealthCheck() {
         return result;
     }
 
+    default CompletableFuture<Void> submitToMigrateReplicas(long ledgerId, String bookieIds)
+            throws UnsupportedEncodingException {
```

Comment:
why we need this exception? If we use **UTF-8**, I think we can remove this throw

Reply:
@Shoothzj PTAL,thanks!